manifest.json

```diff
@@ -1,4 +1,6 @@
 {
+
```

Comment:
We have frozen the `manifest.json` file and don't accept any change in it. That's why the "Manifest Backward Compatibility Check" failed. Please remove this change from the PR.